integration-tests/tests.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"github.com/CESARBR/knot-cloud-storage/pkg/entities"
+)
+
+func showQueryResult(output []entities.Data) {
+	fmt.Printf("\n\n")
+	fmt.Println("THING_ID SENSOR_ID VALUE TIMESTAMP")
+	for i := range output {
+		fmt.Println(output[i])
+	}
+}
+
+func startTest(msg error, testFunction error) {
+	status := "PASS"
+	err := testFunction
+	if err != nil {
+		status = "FAIL"
+	}
+	log.Printf("%s: %s", status, msg)
+
+}
+
+func validateOutputQuery(output []entities.Data, payload *thing) error {
+	if (output[len(output) - 1].From != payload.ID) && (output[len(output) - 1].Payload.SensorID != payload.Data[len(payload.Data) - 1].SensorID) && (output[len(output) - 1].Payload.Value != payload.Data[len(payload.Data) - 1].Value) {
+		return errQueryResult
+	}
+
+	if (output[len(output) - 1].From == payload.ID) && (output[len(output) - 1].Payload.SensorID != payload.Data[len(payload.Data) - 1].SensorID) && (output[len(output) - 1].Payload.Value != payload.Data[len(payload.Data) - 1].Value) {
```

Comment:
I think this isn't a good way to validate if the payload is in the data array. The right approach would be to iterate over the `output` and compare its payload with the one passed as the second argument. Please, take a look [here](https://www.geeksforgeeks.org/structure-equality-in-golang/#:~:text=In%20Go%20language%2C%20you%20are,operator%20or%20DeeplyEqual()%20Method.) and [here](https://stackoverflow.com/questions/24534072/how-to-compare-if-two-structs-slices-or-maps-are-equal).

Reply:
Done. I removed the if's. Check if it's OK, because I still believe it isn't necessary to iterate over the `output`, only check its last element.